src/pdm/cli/commands/venv/backends.py

```diff
@@ -43,7 +43,10 @@ def match_func(py_version: PythonInfo) -> bool:
                 py_version.valid and self.project.python_requires.contains(py_version.version, True)
             )
 
-        for py_version in self.project.iter_interpreters(self.python, search_venv=False, filter_func=match_func):
+        respect_version_file = self.project.config["python.use_python_version"]
```

Comment:
why only pass the config value in venv backend? IIRC there are other references of `iter_interpreters()`